testsuite/fixtures/native_index/ma/make/make-external.toml

```diff
@@ -1,9 +1,9 @@
-[general]
 description = "Utility for directing compilation"
+name = "make"
+licenses = []
 maintainers = ["alejandro@mosteo.com"]
 maintainers-logins = ["mylogin"]
-licenses = []
 
 [[external]]
 kind = "system"
-origin = [] # No package name since we require it unavailable in tests
+origin = []
```

Comment:
The comment should probably be preserved here.

Reply:
Oh right. I'm now concerned about other comments we will have lost. I will check it out.

Reply:
Barring error, I have recovered all meaningful comments in the test suite and community index